### PR TITLE
initial excel work

### PR DESCRIPTION
**Pull request by morellaj**

### CODE REVIEW

File: Main/DataRow.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        Date = date;
+        Hours = hours;
+        Code = code;
+      }
+
+      public string EmployeeNumber;
+      public string Action;
+      public string Date;
+      public string Hours;
+      public string Code;
```
Comment (arielleferre23) [inline Main/DataRow.cs]: Usually the declaration of variables happens at the beginning of the class

File: Main/Program.cs [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string path = "C:\\excel.xlsx";
+            ExcelPackage.LicenseContext = LicenseContext.Commercial;
+            EmployeeData data = new EmployeeData(path);
+            var sheet = data.sheet;
```
Comment (arielleferre23) [inline Main/Program.cs]: I think it's best to avoid 'var' whenever possible

File: Main/Program.cs [hunk 3]
```diff
@@ -1,12 +1,21 @@
 ﻿using System;
+using OfficeOpenXml;
+using System.IO;
+using ETL;
 
 namespace Main
```
Comment (arielleferre23) [inline Main/Program.cs:3]: Some .Net pro can correct me if I'm wrong here, but I would not even use this Main namespace and just place everything under ETL

File: ETL/DTO/DataRow.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
+using System.IO;
+using System;
+using OfficeOpenXml;
+using System.Globalization;
+
+
+namespace ETL
```
Comment (arielleferre23) [inline ETL/DTO/DataRow.cs]: To match the folder structure, might want to change the namespace to ETL.DTO

File: ETL.Tests/DataRow/WhenCreated.cs [hunk 5]
```diff
@@ -0,0 +1,53 @@
+using System;
+using Xunit;
+using System.Globalization;
+
+
+namespace ETL.Tests.DataRowTests
+{
+  public class WhenCreated
```
Comment (moondc) [inline ETL.Tests/DataRow/WhenCreated.cs:8]: Classes should be nouns

File: ETL/EmployeeObj.cs [hunk 6]
```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+
+namespace ETL
+{
+  public class EmpObj
+  {
+    public int empNum;
```
Comment (moondc) [inline ETL/EmployeeObj.cs]: naming convention says that everything is PascalCase except for parameters which are camelCase

File: Main/Program.cs (hunk 3 above)
Comment (moondc) [reply Main/Program.cs:3]: I'm not aware of any benefits for doing this

File: ETL/EmployeeObj.cs [hunk 7]
```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+
+namespace ETL
+{
+  public class EmpObj
```
Comment (arielleferre23) [inline ETL/EmployeeObj.cs]: I think the class should match the name of the cs file

File: ETL/DTO/EmployeeData.cs [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    public ExcelWorksheet worksheet;
+    public List<int> empStartRowIndexList;
+
+    public EmpDataParser(string path)
+    {
+      worksheet = GetWorksheet(path);
+      empStartRowIndexList = GetEmpStartRowIndices();
+    }
+
+    public List<EmpObj> GetAllEmpData()
```
Comment (moondc) [inline ETL/DTO/EmployeeData.cs]: DTO's do not have methods.  They have a constructor and read only properties

**Pull request closed without merging**